frontend/src/lib/components/LemonDivider/LemonDivider.tsx

```diff
@@ -5,19 +5,35 @@ import './LemonDivider.scss'
 export interface LemonDividerProps {
     /** Twice the default amount of margin. */
     large?: boolean
+    /** 3x the thickness of the line. */
+    thick?: boolean
     /** Whether the divider should be vertical (for separating left-to-right) instead of horizontal (top-to-bottom). */
     vertical?: boolean
+    /** Whether the divider should be a dashed line. */
+    dashed?: boolean
```

Comment:
Non-blocking, but would be amazing to have this documented in Storybook too

Reply:
Done!